### DISCUSSION TITLE
编辑游戏简介时回车会直接启动游戏

### DISCUSSION THREAD

**Discussion by Eostrehold** [BakaXL-Support/BakaXL-QA-docs#135] [使用时的疑问]
![image](https://github.com/BakaXL-Support/BakaXL-QA-docs/assets/125660377/012c7844-e637-441e-94f4-1dd5d80270a3)
这个页面，以及更新启动器会使其简介、作者、游玩时间等内容丢失

**Comment by XiaoMouz** [comment]
应该是bug
可以提issue到 BakaXL的仓库
